recipes/xxsds-sdsl-lite/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package ${CONAN_LIBS})
```

```suggestion
conan_basic_setup(TARGETS)

find_package(sdsl-lite CONFIG REQUIRED)

add_executable(test_package test_package.cpp)
target_link_libraries(test_package sdsl-lite::sdsl-lite)
```

So we can validate the component name